ext/mysqli/tests/functions/mysqli_errno.phpt

```diff
@@ -0,0 +1,37 @@
+--TEST--
+function test: mysqli_errno
+--EXTENSIONS--
+mysqli
+--SKIPIF--
+<?php
+require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
+mysqli_check_skip_test();
+?>
+--FILE--
+<?php
+require_once dirname(__DIR__) . "/test_setup/test_helpers.inc";
+
+/* Disable exceptions */
+mysqli_report(MYSQLI_REPORT_OFF);
+
+    $link = default_mysqli_connect();
+    $errno = mysqli_errno($link);
+    var_dump($errno);
+
+    mysqli_query($link, "SELECT * FROM non_existing_table");
+    $errno = mysqli_errno($link);
+
+    var_dump($errno);
+
+    mysqli_close($link);
+    print "done!";
+?>
+--CLEAN--
+<?php
+// Re-enable exceptions
+mysqli_report(MYSQLI_REPORT_ERROR | MYSQLI_REPORT_STRICT);
+?>
```

Comment:
Correct me if I am wrong, but I thought CLEAN sections are sandboxed, so this is unnecessary. This setting lasts only per execution of the script.

Reply:
I actually don't know if they are sandboxed, so you might be correct

Reply:
Just been told they do, I will remove this